pkg/controller/secret/secret_controller.go

```diff
@@ -241,6 +241,9 @@ func createPagerdutyRoute(namespaceList []string) *alertmanager.Route {
 
 		// https://issues.redhat.com/browse/OSD-8689
 		{Receiver: receiverNull, Match: map[string]string{"alertname": "CsvAbnormalFailedOver2Min", "exported_namespace": "redhat-ods-operator"}},
+
+		// https://issues.redhat.com/browse/OSD-9062
+		{Receiver: receiverNull, Match: map[string]string{"severity": "alert"}},
```

Comment:
Is there anyway we can inverse this to != "warning/critical/info" ?

Reply:
Golang doesn't support negative lookarounds in its regex matching, so no, not easily.